_posts/2021-12-9-arrow-duckdb.md

```diff
@@ -0,0 +1,381 @@
+---
+layout: post
+title: "DuckDB quacks Arrow: A zero-copy data integration between Arrow and DuckDB"
+date: "2021-12-9 00:00:00"
+author: Pedro Holanda, Jonathan Keane
+categories: [application]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+*TLDR: The zero-copy integration between DuckDB and Apache Arrow allows for rapid analysis of larger than memory datasets in Python and R using either SQL or relational APIs.*
+
```

Comment:
```suggestion

This post is a collaboration with and cross-posted on [the DuckDB blog](https://duckdb.org/2021/12/03/duck-arrow.html) as well

```

Reply:
Also added a note

Reply:
Will update the link once yours is known